workos/resources/list.py

```diff
@@ -121,7 +136,7 @@ def __iter__(self) -> Iterator[ListableResource]:  # type: ignore
                 if after is not None:
                     next_page = self.list_method(
                         after=after, **fixed_pagination_params, **filter_params
-                    )
+                    )  # type: ignore
```

Comment:
We should add some context when we add a `# type: ignore`

Reply:
Good point, adding.